apps/backend/src/resolvers/types/ExperimentSafety.ts

```diff
@@ -0,0 +1,117 @@
+import { GraphQLError } from 'graphql';
+import { container } from 'tsyringe';
+import {
+  Ctx,
+  Directive,
+  Field,
+  FieldResolver,
+  ObjectType,
+  Resolver,
+  Root,
+} from 'type-graphql';
+
+import { Tokens } from '../../config/Tokens';
+import { ResolverContext } from '../../context';
+import { ExperimentDataSource } from '../../datasources/ExperimentDataSource';
+import { ExperimentSafety as ExperimentSafetyOrigin } from '../../models/Experiment';
+import { ExperimentHasSample } from './ExperimentHasSample';
+import { Proposal } from './Proposal';
+import { Questionary } from './Questionary';
+
+@ObjectType()
+@Directive('@key(fields: "experimentSafetyPk")')

```

Comment:
question: This means that we also need to update Scheduler, do we need to make a PR on scheduler as well before this gets merged?